core/src/main/scala/org/apache/spark/deploy/JsonProtocol.scala

```diff
@@ -44,7 +44,7 @@ private[deploy] object JsonProtocol {
     ("starttime" -> obj.startTime) ~
     ("id" -> obj.id) ~
     ("name" -> obj.desc.name) ~
-    ("cores" -> obj.desc.maxCores) ~
```

Comment:
@nanop this is to align with Master UI. I can update the PR to add additional field in the JSON API if confirmed from maintainers.